Documentation/LegacyAI/PrivateWorkbookConversion.workbook

```diff
@@ -42,6 +42,19 @@
             },
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "3872fc90-1467-4b01-81ef-d82d90665d72",
+            "version": "KqlParameterItem/1.0",
+            "name": "Category",
+            "type": 2,
+            "description": "Workbook Category",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "jsonData": "[\"workbook\", \"sentinel\", \"usage\"]",
```

Comment:
there's a lot more though, technically: 
```
export const WorkbookTypes = {
    default: "workbook",
    tsg: "tsg",
    usage: "usage",
    usageMetrics: "usageMetrics",
    workItems: "workItems",
    performanceWebSites: "performance-websites",
    performanceInsightsComponents: "performance-appinsights",
    performanceDocumentDb: "performance-documentdb",
    performanceStorage: "performance-storage",
    performanceStorageClassic: "performance-storageclassic",
    performanceVM: "performance-vm",
    performanceVMClassic: "performance-vmclassic",
    performanceSqlServersDatabases: "performance-sqlserverdatabases",
    performanceNetworkVirtualNetworks: "performance-virtualnetwork",
    performanceVirtualMachineScaleSets: "performance-virtualmachinescalesets",
    performanceComputeDisks: "performance-computedisks",
    performanceNetworkInterfaces: "performance-networkinterfaces",
    performanceLogicWorkflows: "performance-logicworkflows",
    performanceAppServicePlans: "performance-appserviceplans",
    performanceNetworkApplicationGateway: "performance-applicationgateway",
    performanceAutomationAccountsRunbooks: "performance-runbooks",
    performanceServiceBusQueues: "performance-servicebusqueues",
    performanceDevicesIotHubs: "performance-iothubs",
    performanceNetworkRoutetables: "performance-networkroutetables",
    performanceCognitiveServicesAccounts: "performance-cognitiveserviceaccounts",
    performanceContainerServiceManagedClusters: "performance-containerservicemanagedclusters",
    performanceServiceFabricClusters: "performance-servicefabricclusters",
    performanceCacheRedis: "performance-cacheredis",
    performanceEventHubNameSpaces: "performance-eventhubnamespaces",
    performanceHdinsightClusters: "performance-hdinsightclusters",
    failureWebSites: "failure-websites",
    failureInsightsComponents: "failure-appinsights",
    failureDocumentDb: "failure-documentdb",
    failureStorage: "failure-storage",
    failureStorageClassic: "failure-storageclassic",
    failureVM: "failure-vm",
    failureVMClassic: "failure-vmclassic",
    failureSqlServersDatabases: "failure-sqlserverdatabases",
    failureNetworkVirtualNetworks: "failure-virtualnetwork",
    failureVirtualMachineScaleSets: "failure-virtualmachinescalesets",
    failureComputeDisks: "failure-computedisks",
    failureNetworkInterfaces: "failure-networkinterfaces",
    failureLogicWorkflows: "failure-logicworkflows",
    failureAppServicePlans: "failure-appserviceplans",
    failureNetworkApplicationGateway: "failure-applicationgateway",
    failureAutomationAccountsRunbooks: "failure-runbooks",
    failureServiceBusQueues: "failure-servicebusqueues",
    failureDevicesIotHubs: "failure-iothubs",
    failureNetworkRoutetables: "failure-networkroutetables",
    failureCognitiveServicesAccounts: "failure-cognitiveserviceaccounts",
    failureContainerServiceManagedClusters: "failure-containerservicemanagedclusters",
    failureServiceFabricClusters: "failure-servicefabricclusters",
    failureCacheRedis: "failure-cacheredis",
    failureEventHubNameSpaces: "failure-eventhubnamespaces",
    failureHdinsightClusters: "failure-hdinsightclusters",
    storageInsights: "storage-insights",
    cosmosDBInsights: "cosmosdb-insights",
    vmInsights: "vm-insights",
    containerInsights: "container-insights",
    keyvaultsInsights: "keyvaults-insights",
    backupInsights: "backup-insights",
    redisCacheInsights: "rediscache-insights",
    serviceBusInsights: "servicebus-insights",
    eventHubInsights: "eventhub-insights",
    workloadInsights: "workload-insights",
    adxClusterInsights: "adxcluster-insights",
    wvdInsights: "wvd-insights",
    activityLogInsights: "activitylog-insights",
    hdiClusterInsights: "hdicluster-insights",
    lawsInsights: "laws-insights",
    sentinel: "sentinel",
    hciInsights: "hci-insights"
};
```
many of these are not directly used anymore so probably won't have any but should still be represented here just in case.  the order you have here, with tsg probably 4th is probably good though.